src/Console/Commands/Game/RoundOpenCommand.php

```diff
@@ -103,5 +111,21 @@ public function handle(): void
         $round = $this->roundFactory->create($roundLeague, $startDate, $realmSize, $packSize, $playersPerRace, $mixedAlignments);
 
         $this->info("Round {$round->number} created in {$roundLeague->key} league, starting at {$round->start_date}. With a realm size of {$round->realm_size} and a pack size of {$round->pack_size}");
+
+        if ($round->mixed_alignment) {
+            // Prepopulate round with 20 mixed realms
+            for($i=1; $i<=20; $i++) {
+                $realm = $this->realmFactory->create($round, 'mixed');
```

Comment:
You don't have to set alignment. It is set to neutral inside the factory if round has mixed.

Reply:
Ahh well it's still a required argument atm

Reply:
Ah.. ofc :D

Reply:
It doesn't make sense for it to be 'mixed', since it gets overwritten if the round has mixed alignments, and 'mixed' isn't even valid (it's supposed to be neutral). I made that string argument nullable in the case of mixed alignments (aka 'neutral').